src/content/en/fundamentals/performance/prpl-pattern/index.md

```diff
@@ -221,6 +222,12 @@ The combination of HTTP/2 and HTTP/2 server push provides the _benefits_ of
 bundling (reduced latency) without actual bundling. Keeping resources separate
 means they can be cached efficiently and be shared between pages.
 
+HTTP/2 Push needs to be utilized with care, as it forces data to the browser,
+even if the file is already in the browser’s local cache or bandwidth is
+already saturated. If done wrong, performance can suffer. [Resource hints]
+might be a good alternative to allow the browser to make smart decisions about
+the prioritization of these requests.  
+
```

Comment:
Can we call out link rel=preload? I think resource hints include enough possible things you could explore that it may be worth getting more concrete with which part we're specifically talking about